### PR TITLE
made about page, created top part of about page

### PR DESCRIPTION
**Pull request by Poojyabadramraju**

### CODE REVIEW

File: src/app/about/AboutContent.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      <div className="flex flex-col md:flex-row items-center justify-around w-full mt-5 p-10">
+        <div className="relative flex-1">
+          <Image src="/about-story.png" alt="about-story-image" objectFit="cover" width={400} height={300}/>
+        </div>
+        <div className="flex flex-1 flex-col items-start gap-5 md:ml-10 mt-8">
+          <SecondaryHeading>Our Story</SecondaryHeading>
+          <p className="text-xl md:text-2xl pl-2">
+            Founded in  2023, Foodie began as a <br/> 
+            small family-owned seafood restaurant. <br/>
+            Over the years, we've expanded our menu <br/>
```
Comment (ketanbaitule) [inline src/app/about/AboutContent.jsx:41]:
```suggestion
            Over the years, we&apos;ve expanded our menu <br/>
```

' -> need to be escaped

File: src/app/menu/page.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import CategoryBox from "@/components/Categories/CategoryBox";
 import TempBox from "@/components/TempBox";
 
-export default function Menu() {
+export default function About() {
```
Comment (ketanbaitule) [inline src/app/menu/page.tsx:4]:
```suggestion
export default function Menu() {
```

No need to change this

File: tailwind.config.ts [hunk 3]
```diff
@@ -15,6 +15,9 @@ const config: Config = {
         backdrop: "rgba(0, 0, 0, 0.4)", // Dark backdrop color
         secondary: "rgba(156 ,163 ,175, 0.5)",
       },
+      boxShadow: {
+        '4xl' : '0 4px 16px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1)',
+      },
```
Comment (ketanbaitule) [inline tailwind.config.ts:20]:
```suggestion

```

This isn't going to repeat so just use shadow-[0 4px 16px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1)]

File: src/app/about/page.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import AboutContent from "./AboutContent";
+
+
+export default function About() {
+    return (
+      <div className="flex flex-col gap-y-5 my-5">
+        <AboutContent/>
```
Comment (ketanbaitule) [inline src/app/about/page.tsx:7]: Directly add the AboutContent in this page, no need for different component

File: src/app/about/AboutContent.jsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        <p className="text-center text-lg md:text-2xl">
+          We are passionate about bringing you the freshest seafood, delectable
+          desserts, mouth-watering burgers, and the best pizzas in town. Our
+          mission is to deliver delicious food that satisfies your cravings and
+          delights your taste buds.
+        </p>
+        <div className="grid grid-cols-11 gap-3 max-sm:gap-2 px-5">
+          {about.map((src, index) => {
+            return (
+              <div key={index} className={`w-14 h-72 max-lg:w-9 max-sm:w-7 relative col-span-1 shadow-4xl shadow-slate-500 ${index===0 ? 'rounded-l-xl':''} ${index===about.length-1 ? 'rounded-r-xl':''}`}>
```
Comment (ketanbaitule) [inline src/app/about/AboutContent.jsx:19]:
```suggestion
              <div key={index} className={`w-14 h-72 max-lg:w-9 max-sm:w-7 relative col-span-1 shadow-[0 4px 16px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1)] shadow-slate-500 ${index===0 ? 'rounded-l-xl':''} ${index===about.length-1 ? 'rounded-r-xl':''}`}>
```

File: src/data/about.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+    "/about3.png",
+    "/about4.png",
+    "/about5.png",
+    "/about6.png",
+    "/about7.png",
+    "/about8.png",
+    "/about9.png",
+    "/about10.png",
+    "/about11.png",
+]
```
Comment (ketanbaitule) [inline src/data/about.js:13]: Add above in about/page.tsx file

Its not going to repeat

**Review by ketanbaitule — CHANGES_REQUESTED** [review]

File: src/app/about/AboutContent.jsx (hunk 1 above)
Comment (ketanbaitule) [reply src/app/about/AboutContent.jsx:41]: Reason for deployment failure

File: src/app/about/AboutContent.jsx (hunk 5 above)
Comment (Poojyabadramraju) [reply src/app/about/AboutContent.jsx:19]: but sir the shadow is not working like this

Comment (ketanbaitule) [reply src/app/about/AboutContent.jsx:19]: Try this: `shadow-[0_4px_16px_-1px_rgb(0,0,0,0.1),0_2px_4px_-2px_rgb(0,0,0,0.1)]`

File: src/app/about/AboutContent.jsx [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+        <div className="relative flex-1">
+          <Image src="/about-story.png" alt="about-story-image" objectFit="cover" width={400} height={300}/>
+        </div>
+        <div className="flex flex-1 flex-col items-start gap-5 md:ml-10 mt-8">
+          <SecondaryHeading>Our Story</SecondaryHeading>
+          <p className="text-xl md:text-2xl pl-2">
+            Founded in  2023, Foodie began as a <br/> 
+            small family-owned seafood restaurant. <br/>
+            Over the years, we've expanded our menu <br/>
+            and our reach, always maintaining our<br/>
```
Comment (ketanbaitule) [inline src/app/about/AboutContent.jsx:42]: Remove the <br> tag

File: src/app/menu/page.tsx (hunk 2 above)
Comment (ketanbaitule) [reply src/app/menu/page.tsx:4]: This is still remaining

Comment (Poojyabadramraju) [reply src/app/menu/page.tsx:4]: oh, Sorry sir I didn't observe that I previously changed /menu page name also, I thought I only did for /about one and thought you were talking about that page only, so changed that one and not /menu page. I will change them as they were now.

**Review by ketanbaitule — APPROVED** [review]

**Pull request merged**